english/tweet-it/tweet-it.md

```diff
@@ -0,0 +1,5 @@
+
+Why try and figure out what could go wrong in a bizzare marshallow-filled room when you have twitter followers to keep happy?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.